modules/swagger-codegen/src/main/resources/csharp/model.mustache

```diff
@@ -13,7 +13,7 @@ namespace {{package}} {
   /// {{description}}
   /// </summary>
   [DataContract]
-  public class {{classname}} {
+  public class {{classname}}{{#parent}} : {{{parent}}}{{/parent}} {
```

Comment:
looks like the changes here came from 5f1df9e, and are introduced as a result of me creating the original PR from `master`.